.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules
+coverage
+dist
+.DS_Store
+
+# these cause more harm than good
```

Comment:
@kentcdodds have had a lot of problems in the past with committing lock files, so that's why he decided to ignore them in all his packages.
Doing the same thing, since he made the decision thoroughly.

Reply:
What kind of problems? I'm interested 🙂 Because Yarn is definitely making a good case explaining the reasons why the lockfiles should be committed.

I'm not too attached to this obviously (the current version doesn't include it), we can roll with it and rethink it if we run into issues later too.

Reply:
I think the main problem is when having 2 separate PRs that add/remove/... dependencies and then you can get conflicts, which is not nice to deal with